docs/architecture/agents.md

```diff
@@ -42,6 +42,15 @@ More specifically, an application agent requires the following resources to exis
 * A Service Account for the agent
 * A RoleBinding that binds the ServiceAccount to the Role
 
+When a ServiceBinding is created on an application namespace, the Application Agent looks for resources mentioned in its specification.
+
+A Role needs to be created which allows to retrive, list and watch ServiceBinding `Application` Resources as Primaza Application Agent runs a dynamic informer for each resource.
```

Comment:
who creates this role? Let be more specific, is it the agent creating this role or the developer

Reply:
Thanks, i think this statement can be removed as we already have that documented in L40. The application agent will be creating this role though.